### DISCUSSION TITLE
Doesn't work with B2A8S20P

### DISCUSSION THREAD

**Discussion by Pemisle** [syssi/esphome-jk-bms#27] [Q&A]
App sees the device and identifies by model , but gives message "Device is not Supported"

**Comment by syssi** [comment]
Please try to contact your seller. May be he can provide some more details about your device.

**Comment by Pemisle** [comment]
Tried JK support, Bought from Hankzor store and waiting for a reply from them.

**Reply by Pemisle** [reply]
Also note I can change the password on the device, but same message when trying to get data:  device is not supported

**Comment by riker65** [comment]
I am using this device, and it is working
